addons/actions/src/lib/util/getPropertiesList.js

```diff
@@ -0,0 +1,18 @@
+const { hasOwnProperty } = Object.prototype;
+
+export default function getPropertiesList(value) {
+  const keys = [];
+
+  // eslint-disable-next-line no-restricted-syntax
+  for (const name in value) {
+    // noinspection JSUnfilteredForInLoop
+    if (
+      keys.indexOf(name) === -1 &&
+      !(typeof value[name] === 'function' && !hasOwnProperty.call(value, name))
```

Comment:
Let's invert the check, so that it's more clear what it does:
`typeof value[name] !== 'function' || hasOwnProperty.call(value, name))`